src/Illuminate/Validation/Rules/ImageFile.php

```diff
@@ -4,20 +4,35 @@
 
 class ImageFile extends File
 {
+    public static $allowSvgByDefault = true;
+
+    public static function allowSvg($allowByDefault = true)
```

Comment:
why not make `allowSvgByDefault(bool)` and `disallowSvgByDefault(bool)`

Reply:
Oversight, will add!